lib/etsy/listing.rb

```diff
@@ -211,6 +217,10 @@ def admirers(options = {})
       (user_ids.size > 0) ? Array(Etsy::User.find(user_ids, options)) : []
     end
 
+    def is_supply
+      !!@result.fetch("is_supply")
```

Comment:
this method is always returns `true`

```
# [22] pry(main)> !!{"is_supply" => "true"}.fetch("is_supply")
# => true
# [23] pry(main)> !!{"is_supply" => "false"}.fetch("is_supply")
# => true
```

Reply:
I didn't realize the API was returning strings rather than booleans here.

If you want to submit a PR, to do something like `@result.fetch("is_supply") == "true"` I'd be happy to merge it.